Tasks/CopyFilesOverSSHV0/copyfilesoverssh.ts

```diff
@@ -69,7 +70,7 @@ function getFilesToCopy(sourceFolder: string, contents: string[]): string[] {
         tl.debug('Include matching ' + pattern);
 
         // let minimatch do the actual filtering
-        const matches: string[] = tl.match(allFiles, pattern, matchOptions);
+        const matches: string[] = minimatch.match(allFiles, pattern, matchOptions);
```

Comment:
That's what this function used to do - https://github.com/microsoft/azure-pipelines-task-lib/blob/325df8b40bfa7f0decf06967fee9ca4e951f359b/node/task.ts#L974